src/utils/count.js

```diff
@@ -0,0 +1,33 @@
+export const getBaseballCount = (computer, user) => {
+  let strike = 0;
+  let ball = 0;
+  let nothing = false;
+
+  // 3스트라이크 early return
+  if (computer === user) {
+    strike = 3;
+    return { strike, ball, nothing };
+  }
+
+  // 카운트
+  for (let i = 0; i < computer.length; i++) {
+    if (computer[i] === user[i]) {
+      strike++;
+      continue;
+    }
+
+    const computerArray = [...String(computer)];
+    const userArray = [...String(user)];
+
+    if (computerArray.includes(userArray[i])) {
+      ball++;
+    }
```

Comment:
여기서 `computerArray`와 `userArray`를 스프레드 연산자로 한번 더 담으신 이유가 있을까요? 그리고 String으로 바꾸신 이유도요!
```suggestion
    if(computer.includes(user[i]){
      ball++;
    }
```
이렇게만 작성해도 간결하고 좋지 않을까 해서요!

Reply:
아 이렇게 하셨네여 뒤에서 ㅎㅎ

Reply:
아마 이전 커밋 내역을 보신 것 같아요!